packages/core/src/discovery.js

```diff
@@ -138,11 +139,24 @@ function processSnapshotResources({ domSnapshot, resources, ...snapshot }) {
   return { ...snapshot, resources };
 }
 
+async function responsiveDomParams(snapshot, client) {
```

Comment:
Not something we want to call per snapshot, move it to the snapshot config validation and cache the response in percy.js so that it can be applied to all snapshots in current build